packages/devtools_app/lib/src/screens/vm_developer/object_inspector/object_viewport.dart

```diff
@@ -70,7 +71,7 @@ class ObjectViewport extends StatelessWidget {
       return 'Instance of ${instance.classRef!.name}';
     }
 
-    return '${object.obj.type} ${object.name ?? '<name>'}';
+    return '${object.obj.type} ${object.name ?? ''}';
```

Comment:
should we call .trim() on this string? if object.name is null, there will be an extra space on the end. here and below